csv-validator-cmd/README.md

```diff
@@ -32,10 +32,6 @@ Usage: validate [options] <csv-path> <csv-schema-path>
         Defines the charset encoding used in the CSV file
   -y <value> | --csv-schema-encoding <value>
         Defines the charset encoding used in the CSV Schema file
-  -i <value> | --integrity-check <value>
-        Defines the filename column in the schema for integrity check
-  -d <value> | --include-directory <value>
-        Specifies whether or not the metadata list the directories for integrity check
```

Comment:
This looks okay but grammar could be improved - 

"Specifies whether or not the metadata list the directories for integrity check" -> "Specifies whether the CSV lists the directories for the integrity check"